src/applications/auth/components/RenderErrorContainer.jsx

```diff
@@ -1,7 +1,7 @@
 import React from 'react';
 import PropTypes from 'prop-types';
 import { AUTH_ERRORS, AUTH_LEVEL } from 'platform/user/authentication/errors';
-import Helpdesk from './HelpdeskContact';
+import ContactCenterInformation from 'platform/user/authentication/components/ContactCenterInformation';
```

Comment:
I noticed the existing component is called `ContactCenterInformation` but the ticket mentions renaming it to `HelpdeskContact`. Not sure if that's actually necessary or not, just thought I would call it out.

Reply:
So, the profile team already has a HelpdeskContact component and I didn't want ours getting confused with theirs. I reached out to Alex but forgot to update the ticket. Thanks for pointing that out!

Reply:
The name ContactCenterInformation was approved by Alex earlier today just for clarification.

Reply:
Yeah that sounds good @CaitHawk also just make sure you move it from a Draft PR to the normal one so that the approval process works as expected.